src/core/startCoreFactory.js

```diff
@@ -50,13 +51,19 @@ function startCoreFactory(
       'core',
       coreCommand,
       [
-        '--publish=20002:20002',
+        `--publish=${config.get('core.rpc.port')}:${config.get('core.rpc.port')}`,
         '--detach',
       ],
     );
 
-    const rpcClient = createRpcClient();
-
+    const rpcClient = createRpcClient(
+      {
+        port: config.get('core.rpc.port'), 
+        user: config.get('core.rpc.user'),
+        pass: config.get('core.rpc.password')
```

Comment:
Let's rename this option to `password`.

Reply:
I don't think we can do that, because [`dashd-rpc`](https://www.npmjs.com/package/@dashevo/dashd-rpc#usage) expects the `config` object to contain a key named `pass`. Or do you mean something else?

Reply:
Got it. Let's keep it.